### ISSUE TITLE
configure R to use with IntelliJ

### ISSUE THREAD

**Issue by mvp5822** [holgerbrandl/r4intellij#203]
Team, 
  I cannot get R to compile in IntelliJ on MacOS.   The following screenshots show what my settings are:
![Screen Shot 2019-08-06 at 2 58 35 PM](https://user-images.githubusercontent.com/53809304/62514734-11b32a80-b85b-11e9-9ed3-e0144ff42875.png)
![Screen Shot 2019-08-06 at 2 58 55 PM](https://user-images.githubusercontent.com/53809304/62514735-14158480-b85b-11e9-9b4d-164a6e5c4417.png)
![Screen Shot 2019-08-06 at 3 00 11 PM](https://user-images.githubusercontent.com/53809304/62514737-1546b180-b85b-11e9-9c8e-5e6a9d1b5656.png)

**Comment by mvp5822** [comment]
my code will compile if I use the R console on the right, but will not function if I code in IntelliJ

**Comment by holgerbrandl** [comment]
R is a scripting language, there is no compilation (unless your doing package development).

All the plugin is doing is to build an index from your installed packages and to provide an embedded terminal to run code selections.

**Comment by mvp5822** [comment]
Thanks for the reply.  Maybe my question is off.  I am preparing for my upcoming class that uses R.  How do I see the output from my script in IntelliJ?  Or do I need to just use R studio for that?

**Comment by holgerbrandl** [comment]
Yes, you could use the integrated console, which will show your results.
